### PR TITLE
added post to database

### PR DESCRIPTION
**Pull request by br-bartel**
Added posting to database and added comments.

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ function Home() {
     const handleSubmit = (event) => {
         event.preventDefault();
         const url = `http://localhost:3001/recipes?food=${query}`;
-
         superagent.get(url)
+        // gets info from the server and parses it into JSON and inserts each object into an array
```
Comment (evanslaton) [inline src/Home.js]: Great comment!

File: src/Home.js [hunk 2]
```diff
@@ -23,18 +23,33 @@ function Home() {
             <h1>Res I.P.</h1>
             <h2>pronounced /ˈresəˌpē/</h2>
             <form>
+                {/* creates a text box and submit button on the same line */}
                 <input onChange={(event) => setQuery(event.target.value)} type="text" name="name" placeholder="i.e. Noodles" />
                 <button onClick={handleSubmit}>search</button>
             </form>
-            {recipes.map(element => <Recipe recipe={element} />)}
+            {/* maps over array and applys the Recipe component to each object in the array of objects */}
```
Comment (evanslaton) [inline src/Home.js]: applies*

File: src/Home.js [hunk 3]
```diff
@@ -23,18 +23,33 @@ function Home() {
...
+            {recipes.map((recipeObject, key) => <Recipe key={key} recipe={recipeObject} />)}
         </div>
     );
 }
 
 function Recipe(props) {
+    const handleSave = (event) => {
+        event.preventDefault();
+        const url = `http://localhost:3001/save`; // database URL
```
Comment (evanslaton) [inline src/Home.js]: This is technically not the database URL, this is actually the URL to our server. The server will then communicate with the database

Comment (br-bartel) [reply src/Home.js]: Thanks for the clarification.

**Pull request merged**